setup.py

```diff
@@ -22,7 +22,7 @@
     license="NASA Open Source Agreement Version 1.3",
     python_requires=">=3",
     install_requires=[
-        "numpy>=1.11.3",
```

Comment:
Pointing this out specifically. Would need to wait for a simupy release and then a couple fixups in simupy-flight for numpy 2 compatibilty.